data/theranamuskan.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "Rana Muskan",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Open Source Contributor, Web Developer, Flutter Developer, Youtuber",
+  "avatar": "https://github.com/theranamuskan.png",
+  "links": [
+    {
+      "name": "Twitter",
+      "url": "https://twitter.com/theranamuskan",
+      "icon": "Fatwitter"
+    },
+    {
+      "name": "Instagram",
+      "url": "https://www.instagram.com/theranamuskan/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/theranamuskan/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "GitHub",
+      "url": "https://github.com/theranamuskan",
+      "icon": "FaGithub"
+    }
+  ],
+  "links": [
+   {
+      "name": "Follow on twitter",
+      "url": "https://twitter.com/theranamuskan",
+      "icon": "Fatwitter"
+   },
+   {
+     "name": "Follow me on Instagram",
+     "url": "https://www.instagram.com/theranamuskan/",
+     "icon": "FaInstagram"
+   }
```

Comment:
This part of your file contains duplicate elements. You can remove this one safely

```suggestion
```